src/components/editor-input/editor-input.stories.jsx

```diff
@@ -93,6 +111,13 @@ export default {
 			},
 		},
 	},
+	decorators: [
```

Comment:
when used decorators, in smaller screens UI is broken - https://d.pr/v/gALosR

Reply:
Done. Thanks 👍🏼